### ISSUE TITLE
Regression: some images are missing when the database is generated with DIGITS

### ISSUE THREAD

**Issue by vdel** [Deepomatic/DIGITS#30]
Example: We have 4000 images in the dataset and ask for 25% validation.
We get:
- 2998 images in the training set
- 999 images in the validation set
